main.py

```diff
@@ -697,7 +697,7 @@ def to_lora(layer):
 def _load_text(root_url, list_doc, is_code=False, join=True):
     list_str = [response.text for file_url in list_doc if (response := requests.get(f'{root_url}/{file_url}')).status_code == 200]
     if is_code:
-        list_str = [f".. code-block:: python\n    :caption: {filename}\n\n{textwrap.indent(code, "    ")}" for filename, code in zip(list_doc, list_str)]
+        list_str = [f".. code-block:: python\n    :caption: {filename}\n\n{textwrap.indent(code, '    ')}" for filename, code in zip(list_doc, list_str)]
```

Comment:
Absolutely! However, I've replaced _load_text() with a simpler load_text() function that simply returns the text content of a file or URL. I apologize once again for not catching your pull request before making my commit.

Reply:
No worries, the new version is clearly better!